File: MPI/mpicheck/tests/functions/1.0/MPI_Type_size.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Datatype var_0;
    int *var_1;
    int ret;
    /* calls */
    ret = MPI_Type_size(var_0, var_1);
    ret = PMPI_Type_size(var_0, var_1);
    return 0;
}

File: MPI/mpicheck/tests/functions/1.0/MPI_Type_size.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2
        call mpi_type_size(var_0, var_1, var_2)
       call pmpi_type_size(var_0, var_1, var_2)
        end program main
    